### PR TITLE
Enables custom number of thumbnails per row

### PR DESCRIPTION
**Pull request by tejovanthn**
Adds a new field on the GUI to input the number of thumbnails per row (default is 4).

### CODE REVIEW

File: official/selection_to_pdf.lua [hunk 1]
```diff
@@ -94,6 +101,14 @@ dt.register_storage("export_pdf","Export thumbnails to pdf",
       if my_title == "" then
         my_title = "Title"
       end
+      local thumbs_per_line = no_of_thumbs_widget.text
+      if thumbs_per_line == "" then
+          thumbs_per_line = 4
+      else
+          thumbs_per_line = tonumber(thumbs_per_line)
+      end
+      width = (1/thumbs_per_line) - 0.01
```
Comment (supertobi) [inline official/selection_to_pdf.lua]: This crashes, if thumbs_per_line = 0. You should add a check.

Comment (chrik5) [inline official/selection_to_pdf.lua]: Or use a slider with hard and soft boundaries? A user can put every string in the entry field. A slider gives back a value.

Comment (supertobi) [inline official/selection_to_pdf.lua]: Yes, that's even better.

**Comment by supertobi** [comment]
@tejovanthn I'd like to push your change. Could you try to add a slider like @chrik5 suggested. You can take a look into the moduleExample.lua line 63 how sliders work:
https://github.com/darktable-org/lua-scripts/blob/master/examples/moduleExample.lua#L63 

Please ask if you need help.

**Comment by supertobi** [comment]
@tejovanthn thanks for the fast change. Could you please rebase your branch. There are at the moment a lot of differences between your branch and the main branch.

**Comment by supertobi** [comment]
OK, it's now merged.

**Comment by tejovanthn** [comment]
:) yaay thanks

Tejovanth N
Alumnus, NITK, Surathkal
(+91) 9902725714

**Pull request merged**